server/interfaces/post.interface.ts

```diff
@@ -1,4 +1,4 @@
-export interface Post {
+export interface IPost {
```

Comment:
Wouldn't you prefer not using the "I" prefix for interfaces? I believe it's in C#'s naming convention